docker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.7
+
+ARG LTD_MASON_VER=0.2.5
+
+RUN apk add --no-cache --update python3 && \
+    pip3 install ltd-mason==${LTD_MASON_VER} --upgrade --no-cache-dir && \
+    rm -rf /root/.cache
+
+ENTRYPOINT ["/usr/bin/ltd-mason"]
```

Comment:
The original `ltd-mason` executable is pretty much non-functional. I'd say either make the default entrypoint use `ltd-mason-travis` or just echo a help statement.

Reply:
That's completely reasonable.  Fixed.